manifests/user.pp

```diff
@@ -39,6 +39,7 @@
 
   if $middleware_ssl or $securityprovider == 'ssl' {
     file { "${homedir}/.mcollective.d/credentials/certs/ca.pem":
+      ensure => 'present',
```

Comment:
`ensure => present` is ambiguous, it'll return true for simlinks and directorys. It's better to do `ensure=>file` :+1: 

See https://github.com/puppet-community/puppet-lint-file_ensure-check